tests/_bootstrap.php

```diff
@@ -40,3 +40,23 @@
 // By default, do not enable the Custom Tables v1 implementation in tests.
 putenv( 'TEC_CUSTOM_TABLES_V1_DISABLED=1' );
 $_ENV['TEC_CUSTOM_TABLES_V1_DISABLED'] = 1;
+
+/**
+ * Ensure coupons are loaded in the order modifiers.
+ *
+ * Modify the global variable directly, because WordPress hasn't loaded at this point.
+ *
+ * @see WP_Hook::build_preinitialized_hooks()
+ */
+if ( ! array_key_exists( 'wp_filter', $GLOBALS ) ) {
+	$GLOBALS['wp_filter'] = [];
+}
+
+$GLOBALS['wp_filter']['tec_tickets_commerce_order_modifiers_coupons_enabled'] = [
+	10 => [
+		[
+			'function'      => '__return_true',
+			'accepted_args' => 1,
+		],
+	],
+];
```

Comment:
Do we need to do this in all the tests ? Shouldn't existing functionality be tested with the false return value as well ?

I would prefer this to be specifically added on the _bootstrap file of its own test suite. There wp has loaded and you can use add_fitler as well